sticht/slack.py

```diff
@@ -29,8 +29,8 @@
 
 try:
     from scribereader import scribereader
-    from clog.readers import construct_conn_msg
-    from clog.readers import find_tail_host
+    from scribereader.clog.readers import construct_conn_msg
+    from scribereader.clog.readers import find_tail_host
```

Comment:
hmm, i think our setup.py here was incorrect to begin with: https://github.com/Yelp/sticht/blob/master/setup.py#L34-L39 doesn't mention scribereader as a dependency ;_;

that said, we should probably add a lower-bound on scribereader to be safe there